lib/linter/linter.js

```diff
@@ -2043,9 +2150,12 @@ class Linter {
             fixedResult,
             fixed = false,
             passNumber = 0,
+            fixPasses = 0,
             currentText = text;
         const debugTextDescription = options && options.filename || `${text.slice(0, 10)}...`;
         const shouldFix = options && typeof options.fix !== "undefined" ? options.fix : true;
+        const stats = options && typeof options.stats !== "undefined" ? options.stats : false;
+        const filename = options && typeof options.filename !== "undefined" ? options.filename : "TEXT";
```

Comment:
We tend to use `"<text>"` for this elsewhere in the codebase. Will that work here?

Also, a bit of cleanup similar to the previous line.

```suggestion
        const filename = options?.filename ?? "<text>";
```